sys/androidmedia/gstamcvideodechybris.c

```diff
@@ -1872,8 +1874,7 @@ gst_amc_video_dec_handle_frame (GstVideoDecoder * decoder,
       switch (idx) {
         case INFO_TRY_AGAIN_LATER:
           GST_DEBUG_OBJECT (self, "Dequeueing input buffer timed out");
-          continue;             /* next try */
-          break;
+          goto dequeue_error;
```

Comment:
The design of the Android API really intends for this to keep going and not raise an error, but if this is more consistent and robust raising an error, then this change should be good.